brainiak/matnorm_models/noise_cov_kron.py

```diff
@@ -0,0 +1,124 @@
+import tensorflow as tf
+import numpy as np
+from .helpers import define_scope
+from .noise_covs import NoiseCovBase
+
+
+def tf_solve_lower_triangular_kron(L, y):
```

Comment:
We are using tf.matrix_triangular_solve() for the individual matrices. Maybe I am not getting what you mean here.